src/prefect/client/orchestration.py

```diff
@@ -3669,6 +3668,61 @@ def create_flow_run(
 
         return flow_run
 
+    def update_flow_run(
```

Comment:
It kinda blows my mind that we never had this before :exploding_head:

Reply:
Oh ignore me, this is for the new _sync_ client, we already had the one for the async client